README.md

```diff
@@ -3,7 +3,7 @@
 This is a minimal boilerplate that should allow a Minted developer to create an App-Renderer compatible application through the microfrontend cdn flow. 
 
 IMPORTANT: Webpack configurations
-This repository has three webpack configs, client, server, and appRendererServer that create their respective bundles. Client and server are required for this repository, while appRendererServer is a specialized bundle that is tailored toward App-r so it will correctly render the serverside bundle. 
+This repository has three webpack configs, client, server and appRendererServer that create their respective bundles. Server is the only bundle required for this repository, as the server bundle builds out the client bundle. appRendererServer is a specialized bundle that is tailored toward App-r so it will correctly render the serverside bundle. 
```

Comment:
The server bundle doesn't not build the client bundle.

What's happening is that the local dev server is running the build for both the server and client as well as providing an Express server to serve the client bundle and server-side render the server bundle.